fingertip_keyboard_input/mouse_and_keyboard_control.py

```diff
@@ -122,6 +124,15 @@ def get_filtered_center(center, t):
 # cap.set(cv2.CAP_PROP_FRAME_HEIGHT, 480)
 # print(f"Attempted to set frame width, currently {str(cap.get(cv2.CAP_PROP_FRAME_WIDTH)),str(cap.get(cv2.CAP_PROP_FRAME_HEIGHT))}")
 
+# check system type
+system_type = platform
+if system_type.lower() == "darwin":
+    system_type = "Mac"
+elif system_type.lower().startswith() == "win":
```

Comment:
Getting the following error:
```
Traceback (most recent call last):
  File "C:\Data\berkeley_classes\assistive-technology-cursor-control\fingertip_keyboard_input\mouse_and_keyboard_control.py", line 131, in <module>
    elif system_type.lower().startswith() == "win":
TypeError: startswith() takes at least 1 argument (0 given)
```

I think this should be `elif system_type.lower().startswith('win'):`